src/picongpu/include/particles/ionization/ionizer/BarrierSuppressionIonization/particleIonizerBSI.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2014 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+
+/* IONIZATION MODEL */
+
+namespace picongpu
+{
+    namespace particleIonizerBSI
+    {
+        template<class Velocity, class Gamma>
+        struct Ionize
+        {
+
+            template<typename EType, typename BType, typename ParticleType >
+                    __host__ DINLINE void operator()(
+                                                        const BType bField, /* at t=0 */
+                                                        const EType eField, /* at t=0 */
+                                                        ParticleType& parentIon) /* with position at t=0 and momentum at t=-1/2 */
+            {
+                
+                /*Barrier Suppression Ionization for hydrogenlike helium */
+                if (math::abs(eField)*UNIT_EFIELD >= 5.14e7 && parentIon[chargeState_] < 2)
```

Comment:
Ah, sure, will do - didn't have so much time for more physics, yet. Didn't see that comment before I pushed again.